proof/refine/ARM/Finalise_R.thy

```diff
@@ -3882,7 +4296,7 @@ lemma replyPop_valid_inQ_queues[wp]:
                     threadGet_wp
          | intro conjI impI)+
   apply (clarsimp simp: obj_at'_def projectKOs)
-  done
+  sorry
```

Comment:
Is there a plan with this sorry? (And some similar sorries which don't have any comments attached to them).

Reply:
This is beyond the scope of my PR, but it changed because of the replyPop spec tweaks.

Reply:
Was a quick fix so I'm it in this PR